flink-runtime/src/main/java/org/apache/flink/runtime/io/network/netty/PartitionRequestServerHandler.java

```diff
@@ -123,6 +124,10 @@ else if (msgClazz == TaskEventRequest.class) {
 
                 outboundQueue.addCreditOrResumeConsumption(
                         request.receiverId, NetworkSequenceViewReader::resumeConsumption);
+            } else if (msgClazz == NettyMessage.AckAllUserRecordsProcessed.class) {
```

Comment:
When modifying the imports I found that for static inner classes, we should be able directly use `import` instead of `import static`, thus I unified all the imports to use `import`.